main.py

```diff
@@ -46,8 +46,8 @@
 
 conv_inputs = 1
 conv_outputs = 256
-num_primary_units = 8
-primary_unit_size = 32 * 6 * 6  # fixme get from conv2d
+num_primary_units = 32 * 6 * 6
+primary_unit_size = 8  # fixme get from conv2d
```

Comment:
i think you're right, i did this part wrong.  let me re-read the paper and consider your change carefully.